pyptex/__init__.py

```diff
@@ -129,54 +129,65 @@
 ```
 """
 
-import sys
-import sympy
-import re
-import os
-import traceback
-import numpy
+from contextlib import suppress
+import datetime
 import glob
+import inspect
+import os
 import pickle
-import time
+import re
+import string
 import subprocess
-import datetime
+import sys
+import time
+import traceback
+from typing import List
 import weakref
-import string
-import inspect
 
-__pdoc__ = {"pyptex.compile":False,
-            "pyptex.generateddir":False,
-            "pyptex.process":False,
-            "pyptex.resolvedeps":False,
-            "pyptex.run":False,}
+import numpy
+import sympy
 
-pypparser = re.compile(r'((?<!\\)%[^\n]*\n)|(@@{)|(@{([^{}]+)}|@{{{(.*?)}}})',re.DOTALL)
 
-__pdoc__["format_my_nanos"] = False
+__pdoc__ = {
+    'pyptex.compile': False,
+    'pyptex.generateddir': False,
+    'pyptex.process': False,
+    'pyptex.resolvedeps': False,
+    'pyptex.run': False,
+}
+
+pypparser = re.compile(r'((?<!\\)%[^\n]*\n)|(@@{)|(@{([^{}]+)}|@{{{(.*?)}}})', re.DOTALL)
+
+
+__pdoc__['format_my_nanos'] = False
 # Credit: abarnet on StackOverflow
-def format_my_nanos(nanos):
+def format_my_nanos(nanos: int):
     """Convert nanoseconds to a human-readable format"""
     dt = datetime.datetime.fromtimestamp(nanos / 1e9)
     return '{}.{:09.0f}'.format(dt.strftime('%Y-%m-%d@%H:%M:%S'), nanos % 1e9)
 
-__pdoc__["dictdiff"] = False
-def dictdiff(A,B):
+
+__pdoc__['dictdiff'] = False
+def dictdiff(A, B):
     A = set(A.items())
     B = set(B.items())
     D = A ^ B
-    if(len(D)==0):
+    if len(D) == 0:
         return None
     return next(iter(D))
 
-__pdoc__["mylatex"] = False
+
+__pdoc__['mylatex'] = False
 def mylatex(X):
-    return sympy.latex(X) if X!=None else ""
+    return sympy.latex(X) if X is not None else ''
```

Comment:
Ah. This reminds me I need to carefully audit all my == and != and figure out whether I really meant to use "is". Thanks for this.